src/component/Navbar.js

```diff
@@ -1,12 +1,15 @@
 import './books.css';
 
 const Navbar = () => (
-  <nav className="">
+  <nav className="navbar">
     <ul className="nav-content">
       <li> BookStore CMS </li>
```

Comment:
- [ ] [REQUIRED] This nav content does not have the exact property as it is on zeplin. Kindly ensure that the color and the font family for the `Bookstore CMS` is applied as it is used on zeplin. The colour is `#0290ff`, the font is `Montserrat-Bold` but you applied `#1f7199` and `Tahoma`. Kindly fix this.

### Your design
![1](https://user-images.githubusercontent.com/58692590/180625675-75d91f68-c652-470e-875e-5cf975d432b9.png)

### Zeplin's design
![2](https://user-images.githubusercontent.com/58692590/180625683-981c515e-397e-43a6-941e-6a700b9380ba.png)